mackup/main.py

```diff
@@ -12,11 +12,12 @@
   mackup --version
 
 Options:
-  -h --help     Show this screen.
-  -f --force    Force every question asked to be answered with "Yes".
-  -n --dry-run  Show steps without executing.
-  -v --verbose  Show additional details.
-  --version     Show version.
+  -h --help               Show this screen.
+  -c FILE --config=FILE   Configuration file to use.
```

Comment:
If we go down this road, I guess we could leverage docopt to set a default file in the docstring, e.g. https://github.com/docopt/docopt#help-message-format